FirebaseInAppMessaging/Sources/Data/FIRIAMFetchResponseParser.m

```diff
@@ -334,7 +334,7 @@ - (FIRIAMMessageDefinition *)convertToMessageDefinitionWithMessageDict:(NSDictio
     }
 
     FIRIAMMessageContentDataWithImageURL *msgData =
-        [[FIRIAMMessageContentDataWithImageURL alloc] initWithMessageTitle:title
+        [[FIRIAMMessageContentDataWithImageURL alloc] initWithMessageTitle:title ?: @""
```

Comment:
Since this is an internal API, would a better alternative be to make the parameter nonnull?

Reply:
> Since this is an internal API, would a better alternative be to make the parameter nonnull?

Do you mean make the api _nullable_?